cli-utils/install-test-dependencies.sh

```
@@ -0,0 +1,158 @@
+    # Unzip the rpk binary to the target directory
+    unzip -o "${FILENAME}" -d $HOME/.local/bin || { echo "Failed to unzip rpk"; exit 1; }
+
+    # Remove the downloaded archive
+    rm "${FILENAME}" || { echo "Failed to remove downloaded archive"; exit 1; }
+
+    # Add the target directory to PATH for the current session
+    export PATH=$HOME/.local/bin:$PATH
+
+    # Add the target directory to PATH for future sessions
+    echo 'export PATH=$HOME/.local/bin:$PATH' >> ~/.bashrc
+    source ~/.bashrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)